pkg/build/netbsd.go

```diff
@@ -66,3 +73,47 @@ func (ctx netbsd) clean(kernelDir string) error {
 	// machine.
 	return nil
 }
+
+// Copy the compiled kernel to the qemu disk image using ssh
+func CopyKernelToDisk(outputDir string) error {
+	temp := []byte(`
+{
+	"qemu": "qemu-system-x86_64",
+	"cpu": 1,
+	"count": 1,
+	"mem": 1024
+}	`)
+	VMconfig := (*json.RawMessage)(&temp)
+	// Create config for booting the disk image
+	cfg := &mgrconfig.Config{
+		Workdir:      outputDir,
+		Image:        filepath.Join(outputDir, "image"),
+		SSHKey:       filepath.Join(outputDir, "key"),
+		SSHUser:      "root",
+		TargetOS:     "netbsd",
+		TargetVMArch: "amd64",
```

Comment:
build function accepts targetArch argument, let's pass it from there.
We will use it only fot "amd64" for now, but it's still better to not hardcode these things.